pkg/controller/siddhiprocess/operator.go

```diff
@@ -270,19 +270,17 @@ func (rsp *ReconcileSiddhiProcess) createArtifacts(
 				sp = rsp.updateErrorStatus(sp, ER, ERROR, "AppDeploymentError", err)
 				continue
 			}
-			if (eventType != controllerutil.OperationResultNone) &&
+			if (eventType == controllerutil.OperationResultCreated) &&
```

Comment:
No need for brackets in if condition. Did you run `gofmt` tool against this file?

Reply:
Fixed in https://github.com/siddhi-io/siddhi-operator/pull/123/commits/5a2fe203135ca70441588c95b228e415d49f3f8a